src/libraries/System.Text.Json/tests/Serialization/ExceptionTests.cs

```diff
@@ -518,12 +518,15 @@ public static void DeserializeUnsupportedType()
             // Any test payload is fine.
             string json = @"""Some string""";
 
-            RunTest<Type>();
-            RunTest<SerializationInfo>();
+            RunTest<Type>(json);
```

Comment:
Per @GrabYourPitchforks'c comment in the parent issue, would it be possible to add tests validating the behaviour for `void*` and other pointer types?

Reply:
Thank you for your comments @eiriktsarpalis.
I found that there had been a test covering the behavior, but the test used only `int*` pointer. 
I suggest it is reasonable to add some more pointer types to `PointerTypes `array. 
Please, let me know which particular types you would like to see there besides `void*`.

https://github.com/dotnet/runtime/blob/16931500e524f332526f4e9e463b2a8ce19a5f16/src/libraries/System.Text.Json/tests/Serialization/InvalidTypeTests.cs#L44-L52
https://github.com/dotnet/runtime/blob/16931500e524f332526f4e9e463b2a8ce19a5f16/src/libraries/System.Text.Json/tests/Serialization/InvalidTypeTests.cs#L142-L147

Reply:
Interesting, I think `int*` should suffice. Thanks for investigating.